org-formation/800-redirects/_tasks.yaml

```
@@ -258,3 +258,21 @@ SynapseDockerRegistryProdDnsForward:
     SourceHostedZoneId: ZHAU99KV4A1WU
     # the value of the CNAME record
     TargetHostName: !CopyValue ['registry-prod-DockerFargateStack-LoadBalancerDNS', !Ref SynapseProdAccount]
+
+
+# forward dev.sagebase.org to dev Signoz ALB in org-sagebase-dnt-dev
+SignozDevAppDnsForward:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.6.9/templates/R53/cname.yaml
+  StackName: !Sub '${resourcePrefix}-signoz-dev-cname'
+  StackDescription: Setup a CNAME for signoz dev ALB
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    Account: !Ref SageITAccount
+  Parameters:
+    # the name of the CNAME record
+    SourceHostName: "dev.sagedpe.org"
+    # ID of the sagedpe.org zone (in sageit account)
+    SourceHostedZoneId: "Z04325181I2YIP983P1AD"
+    # the value of the CNAME record
+    TargetHostName: "a09a38cc5a8d6497ea69c6bf6318701b-1974793757.us-east-1.elb.amazonaws.com"
```

```suggestion
    TargetHostName: "ac5c848ac4ff54e2bb11dd87685375b0-1875694220.us-east-1.elb.amazonaws.com"
```

This is the current value that it should be